examples/ci-for-javascript-repo/gitlab-aws-no-import-map-deployer/.gitlab-ci.yml

```diff
@@ -0,0 +1,84 @@
+image: node
+
+cache:
+  key: ${CI_COMMIT_REF_SLUG}
+  paths:
+    - node_modules/
+
+stages:
+  - setup
+  - build
+  - deploy-dev
+  - deploy-qa
+  - deploy-stage
+  - deploy-prod
+
+install:
+  stage: setup
+  script: yarn install --frozen-lockfile
```

Comment:
you'll need to include an artifacts key in this one, and use it in `build` step right?

Reply:
No - this is a functioning file we use at Midigator. The cache stuff at the top of the file is global and shared between all stages and jobs.